src/opm/parser/eclipse/EclipseState/Tables/Rock2dtrTable.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright (C) 2019 by Norce
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <vector>
+#include <opm/parser/eclipse/Deck/DeckItem.hpp>
+#include <opm/parser/eclipse/Deck/DeckKeyword.hpp>
+#include <opm/parser/eclipse/Deck/DeckRecord.hpp>
+#include <opm/parser/eclipse/EclipseState/Tables/Rock2dtrTable.hpp>
+
+namespace Opm {
+
+        Rock2dtrTable::Rock2dtrTable()
+        {
+        }
+
+        void Rock2dtrTable::init(const DeckRecord& record, size_t tableIdx)
+        {
+            m_pressureValues.push_back(record.getItem("PRESSURE").getSIDoubleData()[0]);
+            m_transMultValues.push_back(record.getItem("TRANSMULT").getSIDoubleData());
```

Comment:
Should we have a check that the TRANSMULT vectors are equally long; i.e. something like this:
```
if (m_transMultvalues.size() > 1) { 
   std::size size = m_transMultvalues.size(); 
   if (m_transMultvalues[size-1].size() != m_transmultvalues[size - 2 ].size())
       throw std::invalid_argument(" ... ");
}
```

Reply:
I similar test is added in opm-simulators (it also checks that all vectors have the same size as in ROCKWNOD.